frontend/src/sub-components/labelScroll.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import styled from "@emotion/styled";
+
+const container = {
+  width: "30%",
+  minWidth: "12rem",
+  height: "5rem",
+  border: "0.2rem solid #136f9f",
+  backgroundColor: "white",
+  color: "black",
+  borderRadius: "0.5rem",
+  fontSize: "1.75rem",
+};
+
+class LabelScroll extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
You don't need this unless you're binding a function or setting a state.

Reply:
alright!